--- taffybar-pristine/taffybar.cabal	2015-04-04 05:49:57.000000000 +0200
+++ taffybar/taffybar.cabal	2015-11-01 17:03:42.771653201 +0100
@@ -56,7 +56,7 @@
   if flag(network-uri)
     build-depends: network-uri >= 2.6 && < 3, network >= 2.6 && < 3
   else
-    build-depends: network-uri < 2.6, network < 2.6
+    build-depends: network < 2.6
   hs-source-dirs: src
   pkgconfig-depends: gtk+-2.0
   exposed-modules: System.Taffybar,
